### PR TITLE
Add Gitpod config

### PR DESCRIPTION
**Pull request by jwpjrdev**
[Gitpod](https://gitpod.io) is a really cool tool and integrating with pest would be nice.

### CODE REVIEW

File: pest/src/error.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ pub struct Error<R> {
     /// Line/column within the input string
     pub line_col: LineColLocation,
     path: Option<String>,
-    line: String,
+    pub line: String,
```
Comment (CAD97) [inline pest/src/error.rs:36]: This change seems unrelated

**Review by CAD97 — CHANGES_REQUESTED** [review]
I'd be happy to merge this modulo one nit

**Comment by jwpjrdev** [comment]
My bad, I was testing some things with that second commit. I forgot GitHub adds them both to the PR 😅

**Comment by jwpjrdev** [comment]
I'll revert that in a sec

**Comment by CAD97** [comment]
bors: r+
